### DISCUSSION TITLE
Elem merge during elemental X/Y

### DISCUSSION THREAD

**Discussion by miasik** [hako-mikan/sd-webui-supermerger#108] [Q&A] [answered]
As I can see in web-ui console, my overrides in "Elemental Merge" windows don't work during "elemental" X/Y run.
I'm in panic :-(

Is there a way to make "Elemental Merge" overrides working for "elemental" X/Y runs too?

**Comment by hako-mikan** [comment]
In XY plot.
Console option for XY plot is must be active.

**Reply by miasik** [reply]
"print change" was activated, but I saw only changes from X/Y's "elemental"

Could you add an option to add whole content of "Elemental Merge" after each line of X/Y's "elemental" please?

**Comment by hako-mikan** [comment]
I got it.
"elemental" is not add to elemental. It replace it.
So I added new type "add elemental". It add XY values you input to elemental box.

**Reply by miasik** [reply]
> It add XY values you input to elemental box.
I can't test right now.
Which of them has the top priority? Does "Elemental Merge" override X/Y's "elemental" or vice versa?

**Comment by hako-mikan** [comment, marked as answer]
XY plot ("elemental") override Elemental merge tab
